engine/src/juliabox/plugins/auth_github/github_auth.py

```diff
@@ -64,6 +64,11 @@ def get(self):
         code = self.get_argument('code', False)
         if code is not False:
             user = yield self.get_authenticated_user(redirect_uri=self_redirect_uri, code=code)
+            if not user:
+                self.rendertpl("index.tpl", cfg=JBoxCfg.nv, state=self.state(
+                    error="Could not retrieve your access token.  Please try again.",
```

Comment:
Will "access token" may be too cryptic for users?
May be: "Github authentication failed." instead?